src/Http/Http.Extensions/src/HttpResponseJsonExtensions.cs

```diff
@@ -29,6 +29,7 @@ public static partial class HttpResponseJsonExtensions
     /// <param name="cancellationToken">A <see cref="CancellationToken"/> used to cancel the operation.</param>
     /// <returns>The task object representing the asynchronous operation.</returns>
     [RequiresUnreferencedCode(RequiresUnreferencedCodeMessage)]
+    [RequiresDynamicCode(RequiresUnreferencedCodeMessage)]
```

Comment:
Same as above - these messages should be specific to "dynamic" code.

Reply:
Can someone remind me exactly what about RequestDelegateFactory that requires unreferenced and dynamic code? Is it only JSON (de)serialization? Or something else? it’s use of Linq.Expressions?

Reply:
* A LOT of reflection to get metadata about the endpoint
* JSON serialization/deserialization
* Expression trees
* MakeGenericType